index.html

```diff
@@ -2,23 +2,29 @@
 <html lang="es">
 <head>
 	<meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="icon" type="image/x-icon" href="assets/icons/logo.svg">
-    <link rel="stylesheet" type="text/css" href="css/normalize.css">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<link rel="icon" type="image/x-icon" href="assets/icons/logo.svg">
+	<link rel="stylesheet" type="text/css" href="css/normalize.css">
 	<link rel="stylesheet" type="text/css" href="css/index.css">
-    <link rel="stylesheet" type="text/css" href="css/grid.css">
-    <link rel="stylesheet" type="text/css" href="css/responsive.css">
+	<link rel="stylesheet" type="text/css" href="css/grid.css">
+	<link rel="stylesheet" type="text/css" href="css/responsive.css">
 	<title>Diario Codo a Codo</title>
 </head>
 <body>
 	<header>
-		<h1>Diario <br> Codo a Codo</h1>
-		<nav>
-			<a href="index.html">Inicio</a>
-			<a href="pages/weather.html">Clima</a>
-			<a href="pages/sports.html">Deportes</a>
-			<a href="">Finanzas</a>
-		</nav>
+		<div class="tittle-navbar">
+			<p>DiarioUno</p>
+			<nav>
+				<a href="index.html">Inicio</a>
+				<a href="pages/weather.html">Clima</a>
+				<a href="pages/sports.html">Deportes</a>
+				<a href="">Finanzas</a>
+			</nav>
+			<button>Registrarse</button>
```

Comment:
`<button>` tag is for forms (specificly for take the input data and then send to the backend). Change `<button>` to an `<input type="button">`